plugins/GSdx/GSdx.cpp

```diff
@@ -417,6 +417,11 @@ void GSdxApp::SetConfig(const char* entry, const char* value)
 	WritePrivateProfileString(m_section.c_str(), entry, value, m_ini.c_str());
 }
 
+int GSdxApp::GetConfig(const char* entry, int value)
```

Comment:
> I think you are supposed to use GetConfigI if you don't want to break the newly introduced config system by gregory...

I don't see how it breaks the new configuration system.

> maybe better set the windows default to GSRendererType::Undefined and when GetConfigI returns undefined overwrite with g_best_renderer?

Then what happens if the game is booted without even opening the GSDX dialog ?